sormas-ui/src/main/java/de/symeda/sormas/ui/travelentry/TravelEntryDataForm.java

```diff
@@ -171,22 +173,24 @@ protected void addFields() {
 		differentPointOfEntryJurisdiction = addCustomField(DIFFERENT_POINT_OF_ENTRY_JURISDICTION, Boolean.class, CheckBox.class);
 		differentPointOfEntryJurisdiction.addStyleName(VSPACE_3);
 
-		Label placeOfStayHeadingLabel = new Label(I18nProperties.getCaption(Captions.travelEntryPointOfEntry));
-		placeOfStayHeadingLabel.addStyleName(H3);
-		getContent().addComponent(placeOfStayHeadingLabel, POINT_OF_ENTRY_HEADING_LOC);
+		Label pointOfEntryHeadingLabel = new Label(I18nProperties.getCaption(Captions.travelEntryPointOfEntry));
+		pointOfEntryHeadingLabel.addStyleName(H3);
+		getContent().addComponent(pointOfEntryHeadingLabel, POINT_OF_ENTRY_HEADING_LOC);
 
 		ComboBox regionCombo = addInfrastructureField(TravelEntryDto.REGION);
 		districtCombo = addInfrastructureField(TravelEntryDto.DISTRICT);
 
 		differentPointOfEntryJurisdiction.addValueChangeListener(e -> {
-			DistrictReferenceDto districtDto;
-			if (differentPointOfEntryJurisdiction.getValue().booleanValue()) {
-				districtCombo.setValue(null);
-				getPointsOfEntryForDistrict(null);
-			} else {
-				cbPointOfEntry.setValue(null);
-				districtDto = (DistrictReferenceDto) responsibleDistrict.getValue();
-				getPointsOfEntryForDistrict(districtDto);
+			if (!ignoreDifferentPointOfEntryJurisdiction) {
+				DistrictReferenceDto districtDto;
```

Comment:
You can combine this definition with the initialisation from line 191 and get rid of this line.